src/utils/exportUtils/Exporter.ts

```diff
@@ -75,9 +75,18 @@ export default abstract class Exporter {
         this.files.push(file);
     }
 
+    protected santizeFileName(filename: string): string {
+        filename = filename.normalize('NFD').replace(/[\u0300-\u036f]/g, "");
+        filename = filename.replace(/[ ]/gim, "-");
```

Comment:
```suggestion
        filename = filename.replace(/\s+/gim, "-");
```

To collect groups of whitespace into a single `-`